doc/Project.toml

```diff
@@ -0,0 +1,5 @@
+[deps]
+Clustering = "aaaa29a8-35af-508c-8bc3-b662a17a0fe5"
+Documenter = "e30172f5-a6a5-5a46-863b-614d45cd2de4"
```

Comment:
It's typical to pin Documenter to a particular version, so you can add
```toml
[compat]
Documenter = "~0.21"
```
or whichever version (>0.19) you choose to the end of this file.